src/v2/guide/deployment.md

```diff
@@ -63,24 +63,24 @@ rollup({
 }).then(...)
 ```
 
-## Pre-Compiling Templates
+## Templates pré-compilés
 
-When using in-DOM templates or in-JavaScript template strings, the template-to-render-function compilation is performed on the fly. This is usually fast enough in most cases, but is best avoided if your application is performance-sensitive.
+Quand vous utilisez des templates dans le DOM ou dans des chaînes de caractère, une compilation des templates vers les fonctions de rendu est exécuté à la volée. C'est habituellement quelque chose d'assez rapide dans la plupart des cas, mais il vaut mieux l'éviter si la performance de vos applications est quelque chose de critique.
 
-The easiest way to pre-compile templates is using [Single-File Components](single-file-components.html) - the associated build setups automatically performs pre-compilation for you, so the built code contains the already compiled render functions instead of raw template strings.
+La manière la plus simple de pré-compiler les templates est d'utiliser les [composants monofichiers](single-file-components.html). Le build de mise en place associé remplit ce rôle pour vous, aussi le code contenu dans le paquetage est déjà compilé en fonction de rendu au lieu d'être un template de chaîne de caractère.
 
-If you are using webpack, and prefer separating JavaScript and template files, you can use [vue-template-loader](https://github.com/ktsn/vue-template-loader), which also transforms the template files into JavaScript render functions during the build step.
+Si vous utilisez webpack, et préférez séparer le JavaScript des fichiers de template, vous pouvez utiliser [vue-template-loader](https://github.com/ktsn/vue-template-loader), qui transforme également les fichiers de template en fonction de rendu pendant la phase de build.
 
-## Extracting Component CSS
+## Extraction CSS depuis les composants
 
-When using Single-File Components, the CSS inside components are injected dynamically as `<style>` tags via JavaScript. This has a small runtime cost, and if you are using server-side rendering it will cause a "flash of unstyled content". Extracting the CSS across all components into the same file will avoid these issues, and also result in better CSS minification and caching.
+Lors de l'utilisation de composants monofichiers, le CSS à l'intérieur du composant est injecté dynamiquement en tant que balise `<style>` via JavaScript. Ceci a un léger coût d'exécution, et si vous utilisez du rendu côté serveur, cela causera le phénomène de FOUC (« flash of unstyled content »). Extraire le CSS depuis tous les composants dans le même fichier évite ces problèmes et mène à une meilleur minification et mise en cache.
```

Comment:
à l'intérieur des composants

sous la forme de balises `<style>`

Flash Of Unstyled Content 
(majuscules pour mettre en avant l'acronyme)

et amène à une meilleure minification et mise en cache du CSS.